keyboards/for_science/keyboard.json

```diff
@@ -14,8 +14,8 @@
     },
     "features": {
         "bootmagic": true,
-        "command": true,
```

Comment:
do not modify files outside of the vial keymap that also exist in upstream qmk, this will cause conflicts during merge